fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/impl/v1/datagen/DatagenEntrypoint.java

```diff
@@ -0,0 +1,5 @@
+package net.fabricmc.fabric.impl.v1.datagen;
+
+public class DatagenEntrypoint {
+	//todo why is there no datagen tutorial for tags?
```

Comment:
Look at the datagen API testmod.

Reply:
I know it exists, but testmods are not documentation